root_script: extract_edges.sbat
Requirement already satisfied: decorator==5.1.0 in /home/casonk/.local/lib/python3.9/site-packages (5.1.0)
WARNING: You are using pip version 21.2.4; however, version 22.2.1 is available.
You should consider upgrading via the '/sw/arcts/centos7/python/3.9.7/bin/python3.9 -m pip install --upgrade pip' command.
Requirement already satisfied: pandas==1.4.2 in /home/casonk/.local/lib/python3.9/site-packages (1.4.2)
Requirement already satisfied: pytz>=2020.1 in /home/casonk/.local/lib/python3.9/site-packages (from pandas==1.4.2) (2022.1)
Requirement already satisfied: python-dateutil>=2.8.1 in /home/casonk/.local/lib/python3.9/site-packages (from pandas==1.4.2) (2.8.2)
Requirement already satisfied: numpy>=1.18.5 in /home/casonk/.local/lib/python3.9/site-packages (from pandas==1.4.2) (1.22.3)
Requirement already satisfied: six>=1.5 in /sw/arcts/centos7/python/3.9.7/lib/python3.9/site-packages (from python-dateutil>=2.8.1->pandas==1.4.2) (1.16.0)
WARNING: You are using pip version 21.2.4; however, version 22.2.1 is available.
You should consider upgrading via the '/sw/arcts/centos7/python/3.9.7/bin/python3.9 -m pip install --upgrade pip' command.

/sw/arcts/centos7/python/3.9.7/bin/python

spinning up at: Wed Aug  3 17:26:37 EDT 2022
Job submitted by gl-login2.arc-ts.umich.edu
Job ID 40361156
Running on gl[3294,3299,3307,3311,3320]
Running in /home/casonk/path/mmani_root/mmani0/shared_data/pushshift_python/Great_Lakes_HPC/pys/insurrection 

0.5 2020-10-04 LEVEL_MAPPED
Traceback (most recent call last):
  File "/gpfs/accounts/mmani_root/mmani0/shared_data/pushshift_python/Great_Lakes_HPC/pys/insurrection/map_communities_util.py", line 45, in <module>
    level_df = pd.read_pickle((id_l + date + ('TRIMMED_DF_{}_{}.pkl').format(j, k)))
  File "/home/casonk/.local/lib/python3.9/site-packages/pandas/io/pickle.py", line 187, in read_pickle
    with get_handle(
  File "/home/casonk/.local/lib/python3.9/site-packages/pandas/io/common.py", line 798, in get_handle
    handle = open(handle, ioargs.mode)
FileNotFoundError: [Errno 2] No such file or directory: '/home/casonk/path/mmani_root/mmani0/shared_data/hot/push_file/IDL/2020-10-04TRIMMED_DF_1_21.pkl'
backing up data to turbo buffer
source := /home/casonk/path/mmani_root/mmani0/shared_data/hot/push_file/IDL/
target := /home/casonk/path/flnt-mmani/Reddit_Communities/push_file/IDL/

176 files to copy

target directory purged

copying *.csv*
cp: cannot stat ‘/home/casonk/path/mmani_root/mmani0/shared_data/hot/push_file/IDL/*.csv*’: No such file or directory

1 of 176 files copied

spinning down at: Wed Aug  3 17:26:51 EDT 2022
